.local/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21-alpine@sha256:eeaab088668869c65a2ee0c2c4df7f8c9920140ede7fba3b777cf5b7e9fdbb69
+FROM golang:1.21-alpine@sha256:55f716237933c85cee01748700755b4ac8736fb1ca974c9aed051691b68d6dc2
```

Comment:
The sha for latest `1.21.5` image is [`101168f254556b660b19fd7c542cde4cdc963053573e67825ed16b8fe20e1ff8`](https://hub.docker.com/layers/library/golang/1.21-alpine/images/sha256-101168f254556b660b19fd7c542cde4cdc963053573e67825ed16b8fe20e1ff8?context=explore)